plugins/processors/converter/README.md

```diff
@@ -39,19 +41,40 @@ will overwrite one another.
     float = []
 ```
 
-### Examples:
+### Example
 
+Convert `port` tag to a string field:
 ```toml
 [[processors.converter]]
   [processors.converter.tags]
     string = ["port"]
+```
+
+```diff
+- apache,port=80,server=debian-stretch-apache BusyWorkers=1,BytesPerReq=0
++ apache,server=debian-stretch-apache port="80",BusyWorkers=1,BytesPerReq=0
```

Comment:
should these be in the same order to be more clear?

Reply:
Yeah I'll update this, it should read better if I move the port to the end of the fields:
```
+ apache,server=debian-stretch-apache BusyWorkers=1,BytesPerReq=0,port="80"
```